examples/python/quickstart_pandas.py

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (2021) The Delta Lake Project Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import delta_sharing
+
+# Point to the profile file. It can be a file on the local file system or a file on a remote storage.
+profile_file = "https://databricks-datasets-oregon.s3-us-west-2.amazonaws.com/delta-sharing/share/open-datasets.share"
+
+# Create a SharingClient.
+client = delta_sharing.SharingClient(profile_file)
+
+# List all shared tables.
+print("########### All available tables #############")
+print(client.list_all_tables())
+
+# Create a url to access a shared table.
+# A table path is the profile file path following with `#` and the fully qualified name of a table (`<share-name>.<schema-name>.<table-name>`).
+table_url = profile_file + "#delta_sharing.default.COVID_19_NYT"
```

Comment:
Could you change this to `owid-covid-data` which is a smaller table? This would make the example script run faster. It's better to print something before `load_as_pandas` to show we are loading, such as:

```
print("########### Loading delta_sharing.default.owid-covid-data as a Pandas DataFrame #############")
```